---
source: src/validation/tests/generic_validation_tests.rs
expression: make_readable(&diagnostics)
---
SyntaxError { message: "Invalid type nature for generic argument. BOOL is no Num.", range: [SourceRange { range: 137..138 }], err_no: type__invalid_nature }
SyntaxError { message: "Invalid type nature for generic argument. BYTE is no Num.", range: [SourceRange { range: 211..212 }], err_no: type__invalid_nature }
SyntaxError { message: "Invalid type nature for generic argument. WORD is no Num.", range: [SourceRange { range: 279..280 }], err_no: type__invalid_nature }
SyntaxError { message: "Invalid type nature for generic argument. DWORD is no Num.", range: [SourceRange { range: 354..355 }], err_no: type__invalid_nature }
SyntaxError { message: "Invalid type nature for generic argument. LWORD is no Num.", range: [SourceRange { range: 423..424 }], err_no: type__invalid_nature }

